docs/endpoints/endpoint_examples.rst

```diff
@@ -1,85 +1,22 @@
-Example Globus Compute Endpoint Configurations
-==============================================
+Example Configurations
+**********************
 
 .. _configuration-section:
 
-Globus Compute has been used on various systems around the world. Below are example configurations
-for commonly used systems. If you would like to add your system to this list please
-contact the Globus Compute Team via Slack.
+While Globus Compute is in use on various systems around the world, getting to a working
+configuration that matches the underlying system constraints and site-administrator's
+requirements often takes trial and error.  Below are example configurations for some
+well-known systems that are known to work.  These serve as a reference and (literally)
+as a copy-paste source for getting started.
```

Comment:
Most of our configs need the user to update user-specific info (allocation id, conda activation string... ). We shouldn't set expectations of a copy-paste working.

Reply:
*nod; I've removed the copy-paste language.